src/OmniSharp.Roslyn.CSharp/Services/Refactoring/RenameService.cs

```diff
@@ -41,9 +44,14 @@ public async Task<RenameResponse> Handle(RenameRequest request)
 
                 if (symbol != null)
                 {
+                    var options = new OmniSharpRenameOptions(
+                        RenameOverloads: _omniSharpOptions.RenameOptions.RenameOverloads,
+                        RenameInStrings: _omniSharpOptions.RenameOptions.RenameInStrings,
+                        RenameInComments: _omniSharpOptions.RenameOptions.RenameInComments);
+
                     try
                     {
-                        solution = await Renamer.RenameSymbolAsync(solution, symbol, request.RenameTo, _workspace.Options);
+                        solution = await OmniSharpRenamer.RenameSymbolAsync(solution, symbol, request.RenameTo, options, nonConflictSymbols: null, CancellationToken.None);
```

Comment:
This seems particularly unfortunate that an API that was entirely public is now having to get external access versions for it -- I assume we have a plan to re-make these public in some form?

Reply:
Possibly at some point...